packages/create-codes/package.json

```diff
@@ -9,8 +9,8 @@
     "templates"
   ],
   "scripts": {
-    "dev": "tsup --watch",
-    "build": "tsup",
+    "dev": "pnpm build -- --watch",
```

Comment:
There are pnpm-related files in the root of the monorepo, but it doesn't explicitly say that `pnpm` is used. It may be helpful to write somewhere (in the monorepo README, for example) that `pnpm` is a requirement.